core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -10031,4 +10031,37 @@ RelNode createLogicalValueRel(RexNode col1, RexNode col2) {
 
     assertThat(toSql(root, DatabaseProduct.BIG_QUERY.getDialect()), isLinux(expectedBigQuery));
   }
+
+
+  @Test public void testParseDateTimeFormat() {
+    final RelBuilder builder = relBuilder();
+    final RexNode parseTSNode1 = builder.call(SqlLibraryOperators.PARSE_DATE,
+        builder.literal("YYYYMMDD"), builder.literal("99991231"));
+    final RexNode parseTSNode2 = builder.call(SqlLibraryOperators.PARSE_TIME,
+        builder.literal("HH24MISS"), builder.literal("122333"));
+    final RexNode parseTSNode3 = builder.call(SqlLibraryOperators.PARSE_TIMESTAMP,
+        builder.literal("YYYYMMDD HH24MISS"), builder.literal("99991231 122333"));
+    final RexNode parseTSNode4 = builder.call(SqlLibraryOperators.PARSE_DATETIME,
```

Comment:
remove parseTSNode3 and parseTSNode4 as parse_timestamp and parse_datetime is already there